interfaces/builtin/audio_playback.go

```diff
@@ -109,6 +110,21 @@ owner /{,var/}run/pulse/** rwk,
 owner /run/pulse/native/ rwk,
 owner /run/user/[0-9]*/ r,
 owner /run/user/[0-9]*/pulse/ rw,
+
+# DBus accesses
+include <abstractions/dbus-strict>
```

Comment:
We are still using the old-style '#include' instead of 'include' in the codebase.